src/ui/pages/Developer.tsx

```diff
@@ -0,0 +1,126 @@
+import { iots, pipe, task, taskEither, taskOption } from '@code-expert/prelude';
+import { BaseDirectory, removeFile } from '@tauri-apps/api/fs';
+import { Alert, Button } from 'antd';
+import { api } from 'api';
+import React from 'react';
+
+import { globalAuthState } from '../../domain/AuthState';
+import { ProjectMetadata } from '../../domain/Project';
+import { createSignedAPIRequest } from '../../domain/createAPIRequest';
+import { Exception, fromError } from '../../domain/exception';
+import { routes, useGlobalContextWithActions } from '../GlobalContext';
+import { VStack } from '../foundation/Layout';
+import { notification } from '../helper/notifications';
+import { deleteLocalProject } from './projects/hooks/useProjectRemove';
+
+// Remove the directory `$APPDATA/users`
+
+export function Developer() {
+  const [, dispatchContext] = useGlobalContextWithActions();
+
+  const testAuth = () => {
+    void pipe(
+      createSignedAPIRequest({
+        path: 'app/checkAccess',
+        method: 'GET',
+        payload: {},
+        codec: iots.strict({ status: iots.string }),
+      }),
+      task.map((a) => {
+        console.log(a);
+        return a;
+      }),
+      taskEither.fold(
+        (e) => {
+          notification.error({ message: e.message, description: 'You are not authorized' });
```

Comment:
using the notification Service from helpers/notifications.ts, this will create a `exception(fromError({ message, description }))`, which is probably not what we want?